app/javascript/mastodon/components/avatar_overlay.tsx

```diff
@@ -18,27 +18,33 @@ export const AvatarOverlay: React.FC<Props> = ({
   baseSize = 36,
   overlaySize = 24,
 }) => {
-  const { hovering, handleMouseEnter, handleMouseLeave } = useHovering(autoPlayGif);
-  const accountSrc = hovering ? account?.get('avatar') : account?.get('avatar_static');
-  const friendSrc = hovering ? friend?.get('avatar') : friend?.get('avatar_static');
+  const { hovering, handleMouseEnter, handleMouseLeave } =
+    useHovering(autoPlayGif);
+  const accountSrc = hovering
+    ? account?.get('avatar')
+    : account?.get('avatar_static');
+  const friendSrc = hovering
+    ? friend?.get('avatar')
+    : friend?.get('avatar_static');
 
   return (
     <div
-      className='account__avatar-overlay' style={{ width: size, height: size }}
+      className="account__avatar-overlay"
```

Comment:
Afaik, this quote style is the opposite of the code style we've been enforcing since the beginning of the project.

Reply:
This can be changed using the [`jsxSingleQuote` option](https://prettier.io/docs/en/options.html#jsx-quotes) if needed, but I think double quotes are the standard nowadays in JSX, as they are in HTML.